tensorflow/core/kernels/BUILD

```diff
@@ -3052,6 +3053,10 @@ tf_kernel_library(
         ":xsmm": ["xsmm_conv2d.h"],
         "//conditions:default": [],
     }),
+    # Override EIGEN_STRONG_INLINE to inline on Windows,
+    # So that it doesn't take 20 minutes to compile conv_grad_ops_3d.cc
+    # and conv_ops_3d.cc. See https://github.com/tensorflow/tensorflow/issues/10521
+    copts = if_windows(["/DEIGEN_STRONG_INLINE=inline"]),
```

Comment:
This has a performance hit on the built binary.
How about adding this on the bazel.rc?
We should have a way to disable this during release builds.
Thats why in cmake build it is not on by default.

Reply:
Sounds good, will do it.